src/containerapp/azext_containerapp/tests/latest/test_containerapp_maintenance_config.py

```diff
@@ -0,0 +1,75 @@
+    def test_containerapp_maintenanceconfig_crudoperations_e2e(self, resource_group):
+        
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+        
+        
+        env_name = self.create_random_name(prefix='aca-maintenance-config-env', length=30)
+        self.cmd('containerapp env create -g {} -n {} --location {}  --logs-destination none --enable-workload-profiles'.format(resource_group, env_name, TEST_LOCATION))
+
+        duration = 10
+        weekday = "Sunday"
+        startHour = 12
+
+        # create a container app environment for a Container App Maintenance Config resource
+        self.cmd('containerapp env show -n {} -g {}'.format(env_name, resource_group), checks=[
+            JMESPathCheck('name', env_name)
+        ])
+
+        ## test for CRUD operations on Maintenance Config
+        self.cmd("az containerapp env maintenance-config add --resource-group {} --environment {} -d {} -w {} -s {}".format(resource_group, env_name, duration, weekday, startHour))
+
+        # verify the resource
+        self.cmd("az containerapp env maintenance-config show --resource-group {} --environment {}".format(resource_group, env_name), checks=[
+            JMESPathCheck('properties.scheduledEntries[0].durationHours', duration),
+            JMESPathCheck('properties.scheduledEntries[0].startHourUtc', startHour),
+            JMESPathCheck('properties.scheduledEntries[0].weekDay', weekday),
+        ])
+
+        updatedDuration = 11
+        updatedWeekday = "Tuesday"
+
+        # update the MaintenanceConfig, check duration and weekday are updated and start hour remains the same
+        self.cmd("az containerapp env maintenance-config update --resource-group {} --environment {} -d {} -w {}".format(resource_group, env_name, updatedDuration, updatedWeekday), checks=[
+            JMESPathCheck('properties.scheduledEntries[0].durationHours', updatedDuration),
+            JMESPathCheck('properties.scheduledEntries[0].startHourUtc', startHour),
+            JMESPathCheck('properties.scheduledEntries[0].weekDay', updatedWeekday),
+        ])
+
+        # update nothing to confirm all properties remain
+        self.cmd("az containerapp env maintenance-config update --resource-group {} --environment {}".format(resource_group, env_name), checks=[
+            JMESPathCheck('properties.scheduledEntries[0].durationHours', updatedDuration),
+            JMESPathCheck('properties.scheduledEntries[0].startHourUtc', startHour),
+            JMESPathCheck('properties.scheduledEntries[0].weekDay', updatedWeekday),
+        ])
+
+        # delete the Container App Maintenance Config resource
+        self.cmd("az containerapp env maintenance-config remove --resource-group {} --environment {} -y".format(resource_group, env_name))
+        
+        self.cmd("az containerapp env maintenance-config show --resource-group {} --environment {}".format(resource_group, env_name), checks=[
```

Comment:
you will return a list for show command? For `show`, please make sure only one resource information is returned. For `list`, you can return a list.

Reply:
Hi @p-bouchon 
Can you change this command to `list`? Because it will return a list and it no name required. Thanks.

Reply:
changed to `list`